mmv1/templates/terraform/examples/edgenetwork_router_with_bgp.tf.tmpl

```diff
@@ -0,0 +1,82 @@
+
+resource "google_edgenetwork_router" "{{$.PrimaryResourceId}}" {
```

Comment:
It looks like the Router resource is not marked as immutable, and a number of resources are therefore presumed to be mutable in-place. If that's the case, this resource will need an [update test](https://googlecloudplatform.github.io/magic-modules/test/test/#add-an-update-test) to verify that those fields can be updated.

Reply:
Can we make an exception and skip this requirement? Due to the dependence on physical hardware, the edgenetwork API can only be tested manually by the internal dev team anyway, so I don't think those sorts of unit tests are useful in this case.